src/i18n/locales/en/translation.json

```
@@ -140,13 +140,22 @@
     "home": {
       "send_satoshis": "⚡️ Send Satoshis ⚡️",
       "recent_transactions": "Recent Transactions",
+      "transaction_list": {
+        "tabs": {
+          "outgoing": "Outgoing",
+          "incoming": "Incoming"
+        }
+      },
       "allowance_view": {
+        "recent_transactions": "Recent Transactions",
         "allowance": "Allowance",
         "sats_used": "sats used",
         "no_transactions": "No transactions on <0>{{name}}</0> yet."
       },
       "default_view": {
-        "no_transactions": "No transactions yet."
+        "recent_transactions": "Recent Transactions",
+        "no_transactions": "No transactions yet.",
```

```suggestion
        "no_transactions": "No transactions made with Alby, yet.",
```

maybe we should indicate that no transactions have been made with Alby?
because in the incoming tab we show all, but in the outgoing tab we only show the ones made with Alby. 
This is somewhat confusing....